Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -163,7 +163,7 @@
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
 from RecoHI.HiTracking.HILowPtConformalPixelTracks_cfi import *
 _highlevelreco_HI_wPixTracks = highlevelreco.copy()
-pp_on_AA_2018.toReplaceWith(highlevelreco, cms.Sequence(_highlevelreco_HI_wPixTracks* hiConformalPixelTracksSequencePhase1))
+(pp_on_XeXe_2017 | pp_on_AA_2018).toReplaceWith(highlevelreco, cms.Sequence(_highlevelreco_HI_wPixTracks* hiConformalPixelTracksSequencePhase1))
```

Comment:
please reshuffle the hiConformalPixelTracksSequencePhase1 addition: move the import of HILowPtConformalPixelTracks as well as the addition of hiConformalPixelTracksSequencePhase1 above before the addition of the centrality.

Reply:
@slava77 thanks a lot. I just made a change